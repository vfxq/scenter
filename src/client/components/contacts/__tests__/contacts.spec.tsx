import * as React from 'react';
import { shallow } from 'enzyme';
import Contacts from '../';

test('CheckboxWithLabel changes the text after click', () => {
  const contacts = shallow(<Contacts  />);

  // Interaction demo
  // expect(checkbox.text()).toEqual('Off');
  // checkbox.find('input').simulate('change');
  // expect(checkbox.text()).toEqual('On');

  // Snapshot demo
  expect(contacts).toMatchSnapshot();
});