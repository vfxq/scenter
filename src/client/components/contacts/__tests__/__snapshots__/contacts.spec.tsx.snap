// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CheckboxWithLabel changes the text after click 1`] = `
<Section>
  <h1>
    Контакты
  </h1>
</Section>
`;
